### DISCUSSION TITLE
Power reduction and RadioMaster hardware.

### DISCUSSION THREAD

**Discussion by inventor7777** [pascallanger/DIY-Multiprotocol-TX-Module#982] [Q&A] [answered]
Hi everyone! I have two quick questions about the MPM in my MT12.

1. Which processor does the MPM have on the RadioMaster MT12? I’m assuming it’s the STM32, but I might be wrong (based on the number of preinstalled protocols)
2. How much power does the module in the MT12 transmit with normally, and how much of a reduction in power does the Low Power switch make? (FCC version)

Thanks all!

**Comment by pascallanger** [comment, marked as answer]
1. STM32
2. Check: https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/7a7b4b2e74d18d97a3b101ce6c1bf994ed44d6af/Multiprotocol/Multiprotocol.h#L707

**Reply by inventor7777** [reply]
Exactly what I needed. Thanks!

**Comment by Jozifek86** [comment]
Hello, I am sorry - my question is out of this line, but I have 2 receivers from Cadet 4 set and, I can not bind together with multimodule. I tryied FY326 protocol, and Pelikan protocol, but not work... Can you help with this? Thanks.

**Reply by pascallanger** [reply]
No idea what a Cadet 4 is... Just try all the protocols not just two... Or ask on a forum if someone knows.

**Comment by Jozifek86** [comment]
Hi man, thanks you for your quick answer. Cadet 4 V3 use receiver with FHSS
protocol and Cadet 4 V3 use FHSS V4 protocol. Receiver is this: RX-602
V4... I see docunentation, there is write, I can use multimodule protocol
FY236, but there is option only fy326. 😉 Or I try Pelikan protocol, byť
binding IS not possible... Thanks.

Dňa st 5. 6. 2024, 10:57 pascallanger ***@***.***> napísal(a):

**Reply by pascallanger** [reply]
Have you really tried  all the sub protocols of Pelikan ? There are 3 of them : https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/master/Protocols_Details.md#sub_protocol-pro---0
If none works then I would need a dump of the original TX with a logic analyzer connected to the RF module component of to send me the original TX.

**Comment by Jozifek86** [comment]
Thanks, I think I do not try all subprotocols... I will. But, Cadet 4
transmitter is  in dome versions. In multiprotocol list I can see protocol
FY236 could be compatible, but somewhere in list is write FY236, and
somewhere FY326... This type of protocol I tryied, too, but withou
success...

Dňa st 5. 6. 2024, 11:31 pascallanger ***@***.***> napísal(a):

**Reply by pascallanger** [reply]
I don't understand why you talk about fy326 this is for sure not the correct protocol... Pelikan is the only one that could match.

**Comment by Jozifek86** [comment]
Hi, because... See picture attached. Just screenshot of site supported
protocols... 🤔

Dňa st 5. 6. 2024, 13:36 pascallanger ***@***.***> napísal(a):

**Reply by pascallanger** [reply]
No picture

**Comment by Jozifek86** [comment]
Where I can send picture?

Dňa st 5. 6. 2024, 14:27 pascallanger ***@***.***> napísal(a):

**Reply by pascallanger** [reply]
Post on github instead of replying to the email. Anyway, there is no mention on FY236 on my github or any relation to Cadet.
https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/master/Protocols_Details.md#fy326---20

**Comment by Jozifek86** [comment]
![Screenshot_2024-06-05-14-13-05-963_com android chrome](https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/assets/157303442/c2ba8c40-b7ee-4816-a1d3-f265b0743c77)

**Comment by benlye** [comment]
Ah, that's on me. I haven't updated the protocol list on multi-module.org in some time. I need to fix that.

**Comment by benlye** [comment]
Actually it looks like I made a copy and paste error there and somehow made up a new protocol. Sorry for the confusion, I'm about to remove it.

**Comment by Jozifek86** [comment]
🤣 so, 1 problém solved! 🤣, now, I have to try binding... Will inform you!
Ok?

Dňa st 5. 6. 2024, 15:09 Ben Lye ***@***.***> napísal(a):

**Comment by Jozifek86** [comment]
Ok... But now, what can I do? Use older version of FW... Like 1.3.3.33 and flash, or wait fór update on actual latest version? Me and my friends would like to bind with our multimodule transmitters 2 kinds of Cadet receivers, could see photos... 1 transmitter IS Radiomaster TX16S running EdgeTx 2.9.3., 1 is Turnigy T9X v2 running er9x v822y i think...
![Screenshot_2024-06-04-07-33-58-342_com android chrome](https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/assets/157303442/218a299e-1ed1-4ce3-85bc-e7ec054a4f87)
![nahradni-prijimac-cadet-4-alpha-beta-skylady-cessna](https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/assets/157303442/f91a8b71-b1aa-47ba-a2ca-9e289f1855e0)

**Reply by pascallanger** [reply]
Come in just try to bind with all sub protocols of Pelikan. That's the only thing you have to do and being asked to do for now 10 emails.

**Comment by Jozifek86** [comment]
Hello, great work a many thanks. I can bind RX-602 v4 with protocol:
Pelikan / type: none.  Nice day!

Dňa st 5. 6. 2024, 16:13 pascallanger ***@***.***> napísal(a):

**Comment by Jozifek86** [comment]
I am sorry, but next question... Multimodul can work with JETI receivers, like DUPLEX R8 atc...? Thnaks!

**Reply by pascallanger** [reply]
If it's not listed it's not supported
